README.md

```diff
@@ -31,11 +31,18 @@ The following pacman operations are extended with AUR capabilities:
 * `-Sc` / `-Scc` (build dir/built packages cache clean)
 * `-Qu` (query upgradeable, `-q` supported)
 
+Pikaur also provides the following additional command line arguments:
```

Comment:
no, it's still not the way how all the options are documented, those couple of flags above are just usage examples, full documentation for flags should be only in the code (ie when --help is passed) and shouldn't be repeated in the readme